src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +15,28 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    public static final String LETTER = "w";
+    public static final String REGEX = "\\W";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> words = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] splitData = line.toLowerCase().split(" ");
+                for (String word : splitData) {
+                    word = word.replaceAll(REGEX, "");
+                    if (word.startsWith(LETTER)) {
+                        words.add(word);
+                    }
+                }
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't ignore the exception, re-throw it